src/gtest/test_merkletree.cpp

```diff
@@ -223,25 +221,29 @@ TEST(merkletree, SaplingVectors) {
 }
 
 TEST(merkletree, emptyroots) {
-    UniValue empty_roots = read_json(MAKE_STRING(json_tests::merkle_roots_empty));
-
     libzcash::EmptyMerkleRoots<64, libzcash::SHA256Compress> emptyroots;
+    std::array<libzcash::SHA256Compress, 65> computed;
 
-    for (size_t depth = 0; depth <= 64; depth++) {
-        expect_test_vector(empty_roots[depth], emptyroots.empty_root(depth));
+    computed.at(0) = libzcash::SHA256Compress::uncommitted();
+    ASSERT_TRUE(emptyroots.empty_root(0) == computed.at(0));
+    for (size_t d = 1; d <= 64; d++) {
+        computed.at(d) = libzcash::SHA256Compress::combine(computed.at(d-1), computed.at(d-1), d-1);
+        ASSERT_TRUE(emptyroots.empty_root(d) == computed.at(d));
     }
```

Comment:
Why does each table contain more roots than we'll use?

(I realise that's what we did before this PR, but I don't see any reason for it.)

Reply:
IIRC it was just to exercise a much larger range than we'd likely need. At the time, we didn't know the exact depth we'd need for Sapling.